data/json/monsters/mammal.json

```
@@ -1571,83 +1295,31 @@
     "vision_night": 6,
     "harvest": "dog_tiny_with_skull",
     "fear_triggers": [ "HURT" ],
-    "delete": { "zombify_into": "mon_zombie_dog" },
+    "delete": { "zombify_into": "mon_zombie_dog", "flags": [ "SWARMS", "GROUP_MORALE" ] },
```

```suggestion
    "zombify_into": " ",
    "delete": { "flags": [ "SWARMS", "GROUP_MORALE" ] },
```

Can you do me a solid and switch the broken syntax for deleting zombification to this while you're here? ``delete`` doesn't work on non-arrays, so they still zombify. You gotta overwrite it with an empty field (not 100% sure I used an empty field in this example, this process is weird)